_sub/compute/helm-atlantis/main.tf

```diff
@@ -104,6 +104,7 @@ resource "kubernetes_secret" "gh" {
 
     data = {
         github_token = var.github_token
+        github_token_flux = var.platform_fluxcd_github_token

```

Comment:
Time to merge these two providers into one, rather than leaking this unnecessary complexity into Atlantis?